util.c

```diff
@@ -787,3 +787,65 @@ char *filter_printf(const char *filter, const char *user) {
   filter_result_len(filter, user, result);
   return result;
 }
+
+size_t rfc4515_length ( const char* in, char* out )
```

Comment:
Please try to follow the coding style in this file a bit more closely, this file typically has no space after opening parantheses.

Reply:
ok